samwise.js

```diff
@@ -1,29 +1,31 @@
-const truncatedString = string.substring(0, maxLength) + (string.length > maxLength ? "..." : "");
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.